sdk/batch/azure-batch/CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Release History
 
+## 12.0.0 (2022-01-15)
+
+### Features
+
+- Added property uploadHeaders to `OutputFileBlobContainerDestination`.
+  - Allows users to set custom HTTP headers on resource file uploads.
+  - Array of type HttpHeader (also being added).
+- Added boolean property `allowTaskPreemption` to `JobSpecification`, `CloudJob`, `JobAddParameter`, `JobPatchParameter`, `JobUpdateParameter`
+  - Mark Tasks as preemptible for higher priority Tasks (requires Comms-Enabled or Single Tenant Pool).
+- Replaced comment (title, description, etc.) references of "low-priority" with "Spot/Low-Priority", to reflect new service behavior.
+  - No API change required.
+  - Low-Priority Compute Nodes (VMs) will continue to be used for User Subscription pools (and only User Subscription pools), as before.
+  - Spot Compute Nodes (VMs) will now be used for Batch Managed (and only Batch Managed pools) pools.
+  - Relevant docs:
+      https://docs.microsoft.com/azure/batch/nodes-and-pools
+    - https://docs.microsoft.com/azure/batch/batch-spot-vms
+
 ## 11.0.1 (Unreleased)
```

Comment:
Assuming this is unused, you should remove this entirely.

Reply:
removed